protocol-designer/src/assets/localization/en/protocol_steps.json

```
@@ -27,7 +27,7 @@
   "duplicate": "Duplicate step",
   "edit_step": "Edit step",
   "engage_height": "Engage height",
-  "final_deck_state": "Final deck state",
+  "final_deck_state": "Ending deck",
```

if there is no specific reason to keep the key name,

```suggestion
  "ending_deck": "Ending deck",
```